guides/plugins/plugins/administration/adding-responsive-behavior.md

```diff
@@ -0,0 +1,52 @@
+# Adding responsive behavior
+
+## Overview
+The Shopware 6 Administration provides two ways of adding classes to elements based on their size, the device helper and the `v-responsive` directive.
+Alternatively you can use `sccs` media queries to make your plugin responsive.
+Learn how to use `scss` here:
+
+{% page-ref page="./add-custom-styles.md" %}
+
+## DeviceHelper
+
+The DeviceHelper provides methods to get device and browser information like the current viewport size.
+The helper methods can be accessed with "this.$device" in every Vue component,
+since it is bound to the Vue prototype.
+
+It provides functionality to run a function on resize to for example add classes to a element.
+The example below shows you how to use the `$device.onResize` helper.
+
+```javascript
+const listener = function (ev) {
+    // do something on resize with the event, like passing  
+};
+const scope = this;
+const component = 'sw-basic-example';
+
+this.$device.onResize({ listener, scope, component });
+this.$device.removeResizeListener(component);
+```
+It also provides many helper functions e.g. to get the screen dimensions.
+Although there are many more as seen below:
+
+| Function                               | Description                 |
+|----------------------------------------|-----------------------------|
+| `this.$device.getViewportWidth();`     | gets the viewport width     |
```

Comment:
Please use capital letters at the beginning of the sentences in the second column